src/components/QuizDetail.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '../stylesheet/Quiz.css';
+
+function QuizDetail() {
+  return (
+    <div>
+      Details Of Quiz
+      <div className="QuizDetail">
+        <div>
+          <div className="QuizDetailText">Min Limit: 1</div>
```

Comment:
these values minlimit, timer limit, max limit questions count should be coming from a constant file rather than coded here. 
Create a quiz.js constant file in constants/ and define these values there

Reply:
Done